app/views/shared/_sponsors.html.erb

```diff
@@ -0,0 +1,61 @@
+ <div class="jumbotron" align="middle">
+  <h1>Proudly supported by</h1>
```

Comment:
I don't think this should be an h1 heading - the level headings should be used to describe structure or hierarchy (i.e. semantic) rather than for stylistic reasons (because we should be handling all the style in our css).